menu/api/src/main/java/team/unnamed/gui/menu/util/PaginatedMenuUtil.java

```diff
@@ -38,7 +38,8 @@ public static <E> Inventory createPage(
                 switch (c) {
                     case 'e': {
                         if (entityIndex >= entitiesSize) {
-                            itemClickable = menuInventory.getItemIfNoEntities().clone(availableSlots.get(entitySlotIndex++));
+                            if (menuInventory.getItemIfNoEntities() != null)
```

Comment:
Maybe you can extract return value of MenuInventory#getItemIfNoEntities to a variable so it's not called everytime, null check cannot be removed unlucky. So it's good, just add the variable.

Reply:
That's true